icu4c/APIChangeReport.md

```diff
@@ -203,7 +336,19 @@ Other existing drafts in ICU 66
 | locid.h |  icu::Locale::ConvertingIterator&lt; Iter, Conv &gt;::ConvertingIterator(Iter, Iter, Conv) | DraftICU 65 | 
 | locid.h |  icu::Locale::Iterator::~Iterator() | DraftICU 65 | 
 | locid.h |  icu::Locale::RangeIterator&lt; Iter &gt;::RangeIterator(Iter, Iter) | DraftICU 65 | 
+| locid.h |  void icu::Locale::canonicalize(UErrorCode&amp;) | 
 | measfmt.h |  void icu::MeasureFormat::parseObject(const UnicodeString&amp;, Formattable&amp;, ParsePosition&amp;) const | DraftICU 53 | 
+| measunit.h |  MeasureUnit icu::MeasureUnit::product(const MeasureUnit&amp;, UErrorCode&amp;) const | 
```

Comment:
These APIs are added; why are they in the pre-existing section?

Reply:
Shouldn't this wait until after the API status update?

Reply:
please recheck

Reply:
regenerated